### DISCUSSION TITLE
[idea] Cli plugin

### DISCUSSION THREAD

**Discussion by ennioVisco** [unicape/use-wagmi#131] [Ideas]
It would be cool to have a `vue` or `use-wagmi` plugin for `@wagmi/cli` that auto-generates hooks like the one that is present for react [@wagmi/cli/react](https://wagmi.sh/cli/api/plugins/react).

What do you think?

The source code of the react plugin can be found here: https://github.com/wevm/wagmi/blob/main/packages/cli/src/plugins/react.ts

**Comment by unicape** [comment]
In plan but lower priority
